virt/release_notes/virt-4-16-release-notes.adoc

```diff
@@ -191,6 +191,10 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 [id="virt-4-16-ki-virtualization"]
 ==== Virtualization
 
+//CNV-43195: 4.16 - VM migration fails with virError
+* VM migrations may fail on clusters with mixed CPU types.
```

Comment:
Would be good to have the bug added in RN.

VM migrations may fail on clusters with mixed CPU types. ( CNV-43195 )